pkg/controller/atlasproject/project.go

```diff
@@ -0,0 +1,44 @@
+package atlasproject
+
+import (
+	"context"
+	"github.com/mongodb-forks/digest"
+	mdbv1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/controller/atlas"
+	"go.mongodb.org/atlas/mongodbatlas"
+	"go.uber.org/zap"
+)
+
+func ensureProjectExists(connection atlas.Connection, project *mdbv1.AtlasProject, log *zap.SugaredLogger) error {
+	// TODO transport + http client + configurable atlas backend!
+	t := digest.NewTransport(connection.PublicKey, connection.PrivateKey)
+	tc, err := t.Client()
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+
+	client, err := mongodbatlas.New(tc, mongodbatlas.SetBaseURL("https://cloud-qa.mongodb.com/api/atlas/v1.0/"))
```

Comment:
[q] `// TODO change?` could this be set via env right now in case someone wants to test with dev or prod?

Reply:
it will be! Just wanted to make this running and avoid a huge PR. 
Operator configuration will be done a proper way later